darshan-util/pydarshan/darshan/experimental/plots/plot_dxt_heatmap.py

```diff
@@ -320,16 +320,18 @@ def plot_heatmap(
     Raises
     ------
 
-    NotImplementedError: raised if "DXT_POSIX" is not in the input modules.
+    NotImplementedError: if a DXT module is not input (i.e. "DXT_POSIX") or if
+    the module is not in the input ``DarshanReport``.
 
     """
-    if "DXT_POSIX" not in mods:
-        # TODO: for the moment reject any cases that don't input "DXT_POSIX"
-        # until we can properly aggregate the data
-        raise NotImplementedError("DXT_POSIX module is required.")
+    if "DXT" not in mod:
+        raise NotImplementedError(f"Only DXT modules are supported.")
```

Comment:
nit: unused f-string

Reply:
I fixed this with the latest round of revisions.